src/esp/assets/ResourceManager.cpp

```diff
@@ -90,6 +92,10 @@ bool ResourceManager::loadScene(const AssetInfo& info,
     // to the void
   }
 
+  // once a scene is loaded, we should have a GL::Context so load the primitives
+  Magnum::Trade::MeshData3D cube = Magnum::Primitives::cubeWireframe();
```

Comment:
Add a:

`namespace Mn = Magnum;`

to the top of the file and use Mn:: instead.

Reply:
Sure, but for consistency we should then update all uses in the file. I'll create a separate PR for this also.